docs/example-apps/rails.md

```diff
@@ -1,45 +1,86 @@
+Create a file in the root of your project with the name `convox.yml` and the following content
+
+Check [convox.yml](https://docs.convox.com/configuration/convox-yml) to see all the possible configurations.
+
+```
+resources: # Here we are creating a database resource to use in our application.
+  database:
+    type: postgres
+
+services:
+  web:
+    build: .
+    port: 3000 # Here we are opening the port in which our applicatin will run. If we need a different port we change it here
+    environment: # Here we define the environment variables that we will use in our application. 
+      - SECRET_KEY_BASE=
+    resources: # Here we are linking our service with the resource we want to use. Since our resource is called "database", Convox will create a DATABASE_URL that can be read by our application.
+      - database
+```
+
+**3.** To avoid Docker from loading unnecessary files you need to define a [.dockerignore](https://docs.docker.com/engine/reference/builder/#dockerignore-file). Create a file in the root of your project with the name `.dockerignore` and the content of our [example](https://github.com/convox-examples/rails/blob/master/.dockerignore).
+**4.** On Rails 6.0+ you need to define any host you will use to access your application. Since Convox generates a host that is different from `localhost` and `0.0.0.0` you need to define it in your application. More information about this configuration [here](https://guides.rubyonrails.org/configuring.html#configuring-middleware).
+
+The URL that Convox generates follows the following format:
+
+```https://[service name].[app name].convox```
+
+In our convox.yml the service name is `web`, thus your URL will be something like:
+
+```https://web.[app name].convox```
+
+Where if you run Convox from the project's application then `app name` will be the folder name. Notice that you can also pass the app name as a parameter i.e: `-a my-app`, in which the host will be:
 
-This file is configured to read database credentials from the `DATABASE_URL` environment variable.
+`https://web.my-app.convox`
+
+**5.** Assuming you want to use postgres in your application, in your gemfile add the following line `gem 'pg'`. If you want a different database you just need to install the gem and define it on the `resources` section of `convox.yml`. More information about the databases we support [here](https://docs.convox.com/reference/primitives/app/resource#types)
+
+
+**6.** As mentioned in step **#2**, when you define a resource Convox will create an environment variable with that name for your application to access it. Given in our scenario the name of our resource is `database`, then it will generate a `DATABASE_URL`, which we will use to store/retrieve data. If you change the name of your resource, remember to also change the name of the variable to `[RESOURCE-name]_URL`. To use Convox' database resource as in our convox.yml, in your `config/database.yml` add the following line under `default`:
```

Comment:
Maybe just link to our documentation on this? https://docs.convox.com/reference/primitives/app/resource#linking